### ISSUE TITLE
set_series_begin is adding dates to the entries after they have been striped

### ISSUE THREAD

**Issue by igorkaldowski** [Flexget/Flexget#1396]
### Expected behaviour:

Series first episode entries to not include dates
### Actual behaviour:

Dates added to series entries
#### Config:

```
 follow_show_from_ep:
    priority: 21
    seen: local
    trakt_list:
      account: '{{secrets.trakt.account}}'
      list: '{{secrets.trakt.new_series}}'
      type: episodes
      strip_dates: yes
    accept_all: yes
    set_series_begin: yes
    list_remove:
      - trakt_list:
          account: '{{secrets.trakt.account}}'
          list: '{{secrets.trakt.series}}'
    list_add:
      - trakt_list:
          account: '{{secrets.trakt.account}}'
          list: '{{secrets.trakt.new_series}}'
          type: episodes

  fix_names:
    priority: 22
    trakt_list:
      account: '{{secrets.trakt.account}}'
      list: '{{secrets.trakt.series}}'
      type: shows
      strip_dates: yes
    manipulate:
      - title:
          replace:            
            regexp: '&'
            format: 'and'
    accept_all: yes
    list_add: 
      - entry_list: trakt_shows


  get_shows:
    priority: 23
    configure_series:
      settings:
        quality: webrip+
        path: '{{secrets.folder.root}}{{secrets.folder.series}}{{tvdb_series_name|default(series_name)|pathscrub}}'
        set:
          series_name: "{{ tvdb_id | replace('&', 'and') }}"
      from:
        entry_list: trakt_shows
    discover:
      what:
        - next_series_episodes: yes
      from:
        - torrentleech: 
            rss_key: '{{secrets.torrentleech.rss}}'
            username: '{{secrets.torrentleech.usr}}'
            password: '{{secrets.torrentleech.pass}}'
    interval: 1 minutes
    transmission:
      host: localhost
      port: 9091
      username: '{{secrets.transmission.user}}'
      password: '{{secrets.transmission.pass}}'
      content_filename: "{{tvdb_series_name|default(series_name)|pathscrub}} - {{tvdb_ep_id|default(series_id)}}{% if tvdb_ep_name|default(False) %} - {{tvdb_ep_name|pathscrub}}{% endif %}{% if quality|default(False) %} - [{{quality}}]{% endif %}"
      path: "/{{secrets.folder.root}}{{secrets.folder.series}}{{tvdb_series_name|default(series_name)|pathscrub}}/"
    pushbullet:
      apikey: '{{secrets.pushbullet.api}}'
      title: "New Episode - {{tvdb_series_name|default(series_name)|pathscrub}}"
      body: "{{tvdb_ep_id|default(series_id)}} - {{quality}}"
```

```

2016-09-07 20:33 INFO     manager                       Test mode, creating a copy from database ...
2016-09-07 20:33 INFO     manager                       Test database created
2016-09-07 20:34 INFO     manager                       Database upgrade is required. Attempting now.
2016-09-07 20:34 INFO     schema                        Plugin api_trakt database is not upgradable. Flushing data and regenerating.
2016-09-07 20:34 VERBOSE  task_queue                    There are 3 tasks to execute. Shutdown will commence when they have completed.
2016-09-07 20:34 VERBOSE  trakt_list    follow_show_from_ep Retrieving `episodes` list `Episodes`
2016-09-07 20:34 VERBOSE  details       follow_show_from_ep Produced 70 entries.
2016-09-07 20:34 VERBOSE  task          follow_show_from_ep ACCEPTED: `Lucifer S02E01 
2016-09-07 20:34 VERBOSE  details       follow_show_from_ep Summary - Accepted: 70 (Rejected: 0 Undecided: 0 Failed: 0)
2016-09-07 20:34 INFO     set_series_begin follow_show_from_ep First episode for "Lucifer (2016)" set to S02E01
2016-09-07 20:34 INFO     list_remove   follow_show_from_ep `trakt_list` is marked as online, would remove accepted items outside of --test mode.
2016-09-07 20:34 INFO     list_add      follow_show_from_ep `trakt_list` is marked as an online plugin, would add accepted items outside of --test mode. Skipping
2016-09-07 20:34 INFO     interval      get_shows       No previous run recorded, running now
2016-09-07 20:34 INFO     transmission  get_shows       Trying to connect to transmission...
2016-09-07 20:34 INFO     transmission  get_shows       Successfully connected to transmission.
2016-09-07 20:34 VERBOSE  next_series_episodes get_shows       Series `Lucifer` with identified_by value `auto` are not supported. 
2016-09-07 20:34 WARNING  discover      get_shows       Input next_series_episodes did not return anything
2016-09-07 20:34 VERBOSE  discover      get_shows       Discovering 0 titles ...
2016-09-07 20:34 INFO     discover      get_shows       Ignoring interval because of --discover-now
2016-09-07 20:34 VERBOSE  details       get_shows       Task didn't produce any entries.
2016-09-07 20:34 VERBOSE  details       get_shows       Summary - Accepted: 0 (Rejected: 0 Undecided: 0 Failed: 0)
2016-09-07 20:34 INFO     manager                       Removed test database
```

**Comment by cvium** [comment]
Usually people use [`next_trakt_episodes`](http://flexget.com/Plugins/next_trakt_episodes) for this

**Comment by igorkaldowski** [comment]
I am trying to set a specific start point rather than just the next episode to be watched. Its going back to the list just while i am testing

**Comment by cvium** [comment]
@igorkaldowski but you are removing from a different list "secrets.trakt.series". So you're essentially just adding the same episode to the same list over and over while also removing the episode from a different list.

Anyway, next_trakt_episodes can also set rather specific starting points.

**Comment by igorkaldowski** [comment]
Thanks for pointing that out. I actually didn't notice that i was removing the episodes from the wrong list. I am trying not to use next_trakt_episodes as i have my media server syncing with trakt so if i set it to next to collect it starts downloading from episode 1

**Comment by gazpachoking** [comment]
The underlying issue here is that trakt_list removes the year from the title if strip_dates is on, but not from the 'series_name' field. set_series_begin looks at that field, rather than the title

**Comment by gazpachoking** [comment]
This is probably fixed by 2f77faf0c2ccd68a4bd45d00666c3bd7c0662758. trakt_list now strips the year from series_name as well when strip_dates is defined. Not 100% sure all the implications for this, we'll see if new issues arise.
